pkg/asset/tls/stock.go

```diff
@@ -18,9 +18,9 @@ const (
 	// KubeCACertName is the filename of the KubeCACert.
 	KubeCACertName = "kube-ca.crt"
 	// EtcdCAKeyName is the filename of the EtcdCAKey.
-	EtcdCAKeyName = "etcd-ca.key"
+	EtcdCAKeyName = "etcd-client-ca.key"
 	// EtcdCACertName is the filename of the EtcdCACert.
-	EtcdCACertName = "etcd-ca.crt"
+	EtcdCACertName = "etcd-client-ca.crt"
```

Comment:
If this is just for clients, can we update the variable names to match?  And possibly update the doc dependency graph and other places using the old naming?

Reply:
I believe that CA is for everything etcd-related now. It's probably more appropriate to change the filenames, but I didn't want to change that yet, since the old codebase assumes the CA is just for clients.

Reply:
> It's probably more appropriate to change the filenames, but I didn't want to change that yet, since the old codebase assumes the CA is just for clients.

Is this the same situation as the kubeconfigs, with [an external dependency][1] relying on the specific path?  Or are these internal references all we'd need to update?

```console
$ git log -1 --oneline origin/master
c53ea04 Merge pull request #309 from crawford/tectonic
$ git grep 'etcd-client-ca\.' origin/master | cat
origin/master:installer/pkg/config-generator/tls.go:	if err := copy.Copy(filepath.Join(clusterDir, etcdCAKeyPath), filepath.Join(clusterDir, "generated/tls/etcd-client-ca.key")); err != nil {
origin/master:installer/pkg/config-generator/tls.go:	if err := copy.Copy(filepath.Join(clusterDir, etcdCACertPath), filepath.Join(clusterDir, "generated/tls/etcd-client-ca.crt")); err != nil {
origin/master:modules/bootkube/resources/bootkube.sh:			--etcd-ca=/assets/tls/etcd-client-ca.crt \
origin/master:modules/bootkube/resources/bootkube.sh:	--cacrt=/opt/tectonic/tls/etcd-client-ca.crt \
origin/master:modules/bootkube/resources/bootkube.sh:	--cakey=/opt/tectonic/tls/etcd-client-ca.key \
origin/master:modules/bootkube/resources/bootkube.sh:		--cacert=/opt/tectonic/tls/etcd-client-ca.crt \
origin/master:modules/bootkube/resources/manifests/kube-apiserver-secret.yaml:  etcd-client-ca.crt: ${etcd_ca_cert}
origin/master:modules/bootkube/resources/manifests/openshift-apiserver-secret.yaml:  etcd-client-ca.crt: ${etcd_ca_cert}
origin/master:pkg/asset/ignition/content/bootkube.go:			--etcd-ca=/assets/tls/etcd-client-ca.crt \
origin/master:pkg/asset/ignition/content/bootkube.go:	--cacrt=/opt/tectonic/tls/etcd-client-ca.crt \
origin/master:pkg/asset/ignition/content/bootkube.go:	--cakey=/opt/tectonic/tls/etcd-client-ca.key \
origin/master:pkg/asset/ignition/content/bootkube.go:		--cacert=/opt/tectonic/tls/etcd-client-ca.crt \
origin/master:pkg/asset/manifests/content/bootkube/kube-apiserver-secret.go:  etcd-client-ca.crt: {{.EtcdCaCert}}
origin/master:pkg/asset/manifests/content/bootkube/openshift-apiserver-secret.go:  etcd-client-ca.crt: {{.EtcdCaCert}}
origin/master:steps/assets/base/ignition-tls.tf:  path = "/opt/tectonic/tls/etcd-client-ca.key"
origin/master:steps/assets/base/ignition-tls.tf:  path = "/opt/tectonic/tls/etcd-client-ca.crt"
origin/master:steps/assets/base/inputs.tf:  etcd_ca_cert_pem                = "${file("${local.tls_path}/etcd-client-ca.crt")}"
origin/master:steps/assets/base/inputs.tf:  etcd_ca_key_pem                 = "${file("${local.tls_path}/etcd-client-ca.key")}"
```

[1]: https://github.com/openshift/installer/pull/310#commitcomment-30611874

Reply:
Some external dependencies:
`etcd-client-ca.crt,key` is expected by the static pods that run bootstrap control plane, (kube-core-operator)

`kubeconfig` (kubeconfig-admin) bootube (binary) expects this file to exist at `auth/kubeconfig` that is used by bootstrap control plane.